### ISSUE TITLE
Berryboot 2.0 and SCSI on synology

### ISSUE THREAD

**Issue by joshikoff** [maxnet/berryboot#520]
Cannot load and store an OS such as PRESSPI to scsi storage on synology. Tried all sort of variants, such as checkfs, first download from standard internet (official Raspian) - they all work fine, however, when trying to install os from USB stick, BerryBoot loops and shows errors on console (see attached file). The USB stick content is ok as I have loaded this OS to another setup with USB disc as storage. Just SCSI fails.
Connection to internet is via ethernet.

Pls advice what and how
![img_3166 1](https://user-images.githubusercontent.com/28642440/43096692-8f26ad8e-8ec2-11e8-9ad6-57c9615bd881.jpg)

**Comment by maxnet** [comment]
The warnings in your screenshot are normal.
They happen when it is figuring out which file system your USB drive has.

Unfortunately, that doesn't give us a clue what is up with your iSCSI.

**Comment by maxnet** [comment]
BTW do note that it can take images from a samba file share as well.

https://www.berryterminal.com/doku.php/berryboot/local_repository_on_network_share

**Comment by joshikoff** [comment]
to further investigate the problem I may send you a logfile. Please tell me which one and where are they located.

**Comment by joshikoff** [comment]
I attach 2 more screenshots believing that they are not normal. In specific, the loading process from USB stick is not working. I reformatted the USB and recopied all OS´s.

Also I tried to load from synology using smb://NAS-IP/directory      (works from a Windows system)

Result is saying that Bootberry cannot find an OS to upload. Maybe i do something completly wrong, it seems to be my fault. Of course. But I dont know how to continue.
I only have Apple iMACs and Synology. Windows 10 runs under Parallels as client machine.

![img_3167](https://user-images.githubusercontent.com/28642440/43124630-5aed3b0a-8f30-11e8-8be7-ea4225176c1d.jpg)
![img_3169](https://user-images.githubusercontent.com/28642440/43124631-5b14e010-8f30-11e8-829b-a57e6a90124a.jpg)
